RDF.js

```diff
@@ -749,14 +749,20 @@ function importItem(newItem, node) {
 		var creatorType = possibleCreatorTypes[i];
 		if(creatorType == "author") {
 			creators = getFirstResults(node, [n.bib+"authors", n.dc+"creator", n.dc1_0+"creator",
-				n.dcterms+"creator", n.eprints+"creators_name",
+				n.dcterms+"creator", n.eprints+"creators_name", n.article+"author", n.video+"director", n.music+"musician",
```

Comment:
Okay, I guess the `n.article+"author"` indicates the Facebook author from above.

I read at the <a href="http://ogp.me/#no_vertical">specification</a> that this is "profile array" where a profile consists of first_name, last_name, user_name, gender. However, I haven't seen such data in any of the examples. And <a href="https://developers.facebook.com/docs/reference/opengraph/object-type/article">here</a> it is written that this is "An array of Facebook profile URLs or IDs of the authors for this article" and that is AFAIS the implementation WordPress supports.

Therefore, I don't see much value in these Facebook links and would suggest to drop `n.article+"author"` completely. I guess it might be similar for the other two properties "director" and "musician".